### DISCUSSION TITLE
Stop Notification Spam?

### DISCUSSION THREAD

**Discussion by jayweaf** [jef/streetmerchant#1547] [Help] [answered]
How do I stop checking an item once it's been found as in stock? It constantly spams notifications and I would like for only one notification to go through and no more until it goes back out of stock again.

**Comment by trillox9** [comment]
I have same problem, maybe page timeout?

**Comment by epatella73** [comment, marked as answer]
hi, you can use the IN_STOCK_WAIT_TIME = " time out in sec." instruction found in the dotenv file
